src/UI/Component/Symbol/Glyph/Factory.php

```diff
@@ -951,4 +951,33 @@ public function time($action = null);
      * @return \ILIAS\UI\Component\Symbol\Glyph\Glyph
      */
     public function close($action = null);
+
+    /**
+     * ---
+     * description:
+     *   purpose: >
+     *      The More Glyph is used to symbolize a set of actions that can not
+     *      be displayed directly due to scarce space.
+     *   composition: >
+     *      The More Glyph uses the glyphicon-option-vertical.
+     *   effect: >
+     *       Clicking the More Glyph displays the actions hidden behind the Glyph.
+     *
+     * context:
+     *    - This Glyph is currently used in the responsive view of the Meta Bar.
```

Comment:
delete this line. In the Meta Bar will be the "Collect Glyph". The "More Glyph" will be visible in the Mainbar only.